dropwizard-views/src/main/java/io/dropwizard/views/ViewMessageBodyWriter.java

```diff
@@ -77,11 +78,11 @@ public void writeTo(View t,
                 }
             }
             throw new ViewRenderException("Unable to find a renderer for " + t.getTemplateName());
-        } catch (FileNotFoundException e) {
-            final String msg = MessageFormat.format(MISSING_TEMPLATE_MSG, t.getTemplateName());
+        } catch (Throwable e) {
```

Comment:
Catching `Throwable` instead of just `Exception` might be too broad here as it also catches non-recoverable errors like `OutOfMemoryError`.